lib/Dancer2/Manual/Migration.pod

```diff
@@ -298,6 +300,26 @@ function with decoded input.
 
 =back
 
+=head3 Templating engine changes
+
+Whereas in Dancer1 it was acceptable to write:
+
+    template '/foo/bar'
+
+In Dancer2, the leading slash is not permitted:
+
+    template 'foo/bar'
+
+Whereas in Dancer1, template engines have the methods:
+
+    $template_engine->view('foo.tt')
+    $template_engine->view_exists('foo.tt')
+
+In Dancer2, you should instead write:
+
+    $template_engine->view_pathname('foo.tt')
+    $template_engine->pathname_exists($full_path)
```

Comment:
I think it's probably fine to put it like that although strictly code where I found this is not writing a templating engine, it's just using TT2. Admittedly, it's wrapping `template` - but all it's really doing is checking to see if the view exists and if not, trying to find another view which does.